### PR TITLE
adds instructions on building a Docker image of this repository

### PR DESCRIPTION
**Pull request by ksatirli**
Hey @tunzor,

@im2nguyen alerted me to this repository, so I decided to use it in demo for [@ksatirli/kubernetes-ingress-with-consul-and-ngrok](https://github.com/ksatirli/kubernetes-ingress-with-consul-and-ngrok/)

For that, I needed a Dockerized version of the repository, so figured I'd share the instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
+
+MAINTAINER Anthony Russo <github.com/tunzor>
+
+RUN apk --update add nginx
+
+COPY ./2048 /usr/share/nginx/html
+
+EXPOSE 80
+
+CMD ["nginx", "-g", "daemon off;"]
```
Comment (im2nguyen) [inline README.md]: this dockerfile works

```suggestion
FROM nginx:latest

COPY . /usr/share/nginx/html

EXPOSE 80

CMD ["nginx", "-g", "daemon off;"]
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,34 @@
...
 
 Playable [here](https://tunzor.ca/hashi-2048)
 
-### Screenshot
+## Screenshot
 ![2048 screenshot](images/screen-cap.jpg)
+
+## Docker Image
+
+A Docker Image of 2048 can be built with the following `Dockerfil`:
```
Comment (im2nguyen) [inline README.md]:
```suggestion
A Docker Image of 2048 can be built with the following `Dockerfile`:
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,34 @@
...
+CMD ["nginx", "-g", "daemon off;"]  
+```
+
+1.) Create a `Dockerfile` in the directory _above_ the clone of this repository.
+
+2.) Build the Docker Image locally using `docker build . --tag "2048"`
+
+3.) Tag the Docker Image using `docker tag 2048 ghcr.io/tunzor/hashi-2048:1.0.0`
+
+4.) Push the Docker Image to [GHCR](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-docker-registry) using `docker push ghcr.io/tunzor/hashi-2048:1.0.0`
```
Comment (im2nguyen) [inline README.md]:
```suggestion
4.) Push the Docker Image to [GHCR](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-docker-registry) using `docker push ghcr.io/tunzor/hashi-2048:1.0.0`

5.) Run the Docker image locally using `docker run -p 8080:80 ghcr.io/tunzor/hashi-2048:1.0.0`
```

File: README.md (hunk 1 above)
Comment (ksatirli) [reply README.md]: This works and is cleaner! I was in a time crunch, so I based it off of the [upstream Dockerfile](https://github.com/alexwhen/docker-2048/blob/master/Dockerfile)

**Pull request closed without merging**